docs/sources/flow/tasks/migrate/from-otelcol.md

```diff
@@ -0,0 +1,276 @@
+---
+aliases:
+- /docs/grafana-cloud/agent/flow/tasks/migrate/from-otelcol/
+- /docs/grafana-cloud/monitor-infrastructure/agent/flow/tasks/migrate/from-otelcol/
+- /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/tasks/migrate/from-otelcol/
+- /docs/grafana-cloud/send-data/agent/flow/tasks/migrate/from-otelcol/
+canonical: https://grafana.com/docs/agent/latest/flow/tasks/migrate/from-otelcol/
+description: Learn how to migrate from OpenTelemetry Collector to Grafana Agent Flow
+menuTitle: Migrate from OpenTelemetry Collector
+title: Migrate from OpenTelemetry Collector to Grafana Agent Flow
+weight: 310
+---
+
+# Migrate from OpenTelemetry Collector to {{% param "PRODUCT_NAME" %}}
+
+The built-in {{< param "PRODUCT_ROOT_NAME" >}} convert command can migrate your [OpenTelemetry Collector][] configuration to a {{< param "PRODUCT_NAME" >}} configuration.
+
+This topic describes how to:
+
+* Convert an OpenTelemetry Collector configuration to a {{< param "PRODUCT_NAME" >}} configuration.
+* Run an OpenTelemetry Collector configuration natively using {{< param "PRODUCT_NAME" >}}.
+
+## Components used in this topic
+
+* [otelcol.receiver.otlp][]
+* [otelcol.processor.memory_limiter][]
+* [otelcol.exporter.otlp][]
+
+## Before you begin
+
+* You must have an existing OpenTelemetry Collector configuration.
+* You must have a set of OpenTelemetry Collector applications ready to push telemetry data to {{< param "PRODUCT_NAME" >}}.
+* You must be familiar with the concept of [Components][] in {{< param "PRODUCT_NAME" >}}.
+
+## Convert an OpenTelemetry Collector configuration
+
+To fully migrate your configuration from [OpenTelemetry Collector] to {{< param "PRODUCT_NAME" >}}, you must convert your OpenTelemetry Collector configuration into a {{< param "PRODUCT_NAME" >}} configuration.
+This conversion will enable you to take full advantage of the many additional features available in {{< param "PRODUCT_NAME" >}}.
+
+> In this task, you will use the [convert][] CLI command to output a {{< param "PRODUCT_NAME" >}}
+> configuration from a OpenTelemetry Collector configuration.
+
+1. Open a terminal window and run the following command.
+
+   {{< code >}}
+
+   ```static-binary
+   AGENT_MODE=flow grafana-agent convert --source-format=otelcol --output=<OUTPUT_CONFIG_PATH> <INPUT_CONFIG_PATH>
+   ```
+
+   ```flow-binary
+   grafana-agent-flow convert --source-format=otelcol --output=<OUTPUT_CONFIG_PATH> <INPUT_CONFIG_PATH>
+   ```
+
+   {{< /code >}}
+
+   Replace the following:
+
+   - _`<INPUT_CONFIG_PATH>`_: The full path to the OpenTelemetry Collector configuration.
+   - _`<OUTPUT_CONFIG_PATH>`_: The full path to output the {{< param "PRODUCT_NAME" >}} configuration.
+
+1. [Run][] {{< param "PRODUCT_NAME" >}} using the new {{< param "PRODUCT_NAME" >}} configuration from _`<OUTPUT_CONFIG_PATH>`_:
```

Comment:
Should we include an example line with the run command and OUTPUT_CONFIG_PATH?

Reply:
We thought about that on the other converter tasks and chose to link to the Run docs since the command can vary quite a bit depending on the environment.